devices/camera.py

```diff
@@ -2570,8 +2570,8 @@ def finish_exposure(
                             hdusmallfits.writeto(
                                 paths["im_path"] + paths["i768sq_name10"] + ".fz"
                             )
-                            if not no_AWS:
-                                g_dev["cam"].enqueue_for_AWS(
+                            if not no_AWS and self.config['send_files_at_end_of_night'] == 'no':
```

Comment:
I think it makes more sense to use a boolean for the `send_files_at_end_of_night` (and `endofnightoverride`) values if the only options are "yes"/"no". This way it won't matter if the capitalization of the string were different, and I think is more clear.